### PR TITLE
Document our Why / How etc

### PR DESCRIPTION
**Pull request by hh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,42 @@
 
 Snooping on the Kubernetes OpenAPI communications
 
+## Welcome to our really new / exploration level project.
+
+### Why
+Our initial goal is to provide a useful indicator as to which Kubernetes APIs are used the most and don't yet have conformance tests. This is specifically to ensure we are creating testing API's that are relevant, rather than just hitting all the endpoints. Our K8s API consumer/user journeys are an important model to drive this prioritization, in addition to providing insight into _how_ the APIs are being used.
```
Comment (AishSundar) [inline README.md]: remove "creating"

Comment (AishSundar) [inline README.md]: nit: s/API's/APIs

**Comment by oomichi** [comment]
Thanks for doing this, @hh :-)
Most part is good for me, it is nice to fix Aish's points.

BTW, I have one question about this kind of thing. Do we want to count all APIs which include ones returned error status codes(like HTTP 404, etc) as tested? or do we want to count succeeded APIs only as we verify happy path of these APIs?

File: README.md [hunk 2]
```diff
@@ -2,6 +2,42 @@
...
+
+Our Secondary goal is a Parallel Certification Program, using same machinery for Certified Kubernetes Provider, to certify set of Apps that utilize the Kubernetes API. ie. Istio, Skaffold, Draft => Require K8s 1.9 If you have 1.9, it will run those tools. If they utilize only v1/stable APIS they are guaranteed to run on at least the next K8s release.
+
+![mockup of what kubernetes conformance could look like](docs/images/conformance_mockup_labels.png "Conformance mockup")
+
+### How
+
+In order to identify target applications to test, we define a Kubernetes API Consumer as a _KAPIC_.
+
+We inspect the [advanced audit logs](https://kubernetes.io/docs/tasks/debug-application-cluster/audit/) to describe which APIs are called during KAPIC operations. For our initial data run we installed a small set of KAPIC helm charts and observing which API groups are called. We used the [d3 libary](https://github.com/d3/d3/wiki/Gallery) to create [Sunburst Partition Graphs](https://bl.ocks.org/mbostock/4063423) center stable/beta/alpha with partitions for APIGroups then APICalls/Verbs.
```
Comment (AishSundar) [inline README.md]: s/observing/observed

**Comment by AishSundar** [comment]
Thanks a lot for doing this @hh and also for the documentation :)

**Comment by hh** [comment]
@oomichi regarding including 404s as tested:

If the e2e / conformance suite is hitting 404s, I think it should be considered a failed test.
As far as the graph of coverage, I'm unsure if the count of endpoints hitting a 404 (from the e2e suite) would be that large. We should dig into the data and see.

If KAPICs are hitting 404s, there is activity at that endpoint but it's likely that application isn't mean to run against that version of k8s. I think the number of 404s we see from applications should also be quite small. A larger data set (more applications) will give us a better understanding here.

We will try and streamline creating and contributing apisnoop data to get us there faster.

**Pull request merged**